gbra/util/ei_graph.py

```diff
@@ -234,7 +240,7 @@ def load(filename):
             possible_ratings = sorted(list(ratings_set))
 
         # Setup the graph with the range of possible ratings.
-        graph.rating_range = (possible_ratings[0], possible_ratings[1])
+        graph.rating_range = (0, 5)
```

Comment:
The rating range for movielens is actually (1, 5). 

The line above actually has a bug.  It should be `graph.rating_range = (possible_ratings[0], possible_ratings[-1])`.